### PR TITLE
refactor: organize SSZ benchmarks

### LINKED ISSUE #987: Organize 'ssz_ex_bench.exs' into specific separate files.
**ssz_ex_bench.exs**  has benchmarks for encoding, decoding and merkleization all together.
We should have a specific file for each topic.

### PR DESCRIPTION
**Pull request by avilagaston9**
This PR organizes SSZ benchmarks.

Closes #987

### CODE REVIEW

File: bench/ssz/ssz_ex_bench.exs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+alias LambdaEthereumConsensus.SszEx
```
Comment (MegaRedHand) [inline bench/ssz/ssz_ex_bench.exs:1]: We can call this one "encode_decode_bench.exs"

File: bench/ssz/merkleize_ex_bench.exs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+alias LambdaEthereumConsensus.SszEx
```
Comment (MegaRedHand) [inline bench/ssz/merkleize_ex_bench.exs:1]: We can drop the "ex" in the name of the benchmark.

File: bench/ssz/ssz_ex_bench.exs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Benchee.run(
+  %{
+    "SszEx.encode" => fn {v} -> SszEx.encode(v) end,
+    "Ssz.to_ssz" => fn {v} -> Ssz.to_ssz(v) end
+  },
+  warmup: 2,
+  time: 5,
+  inputs: %{
+    "BeaconState" => {state},
+    "Checkpoint" => {checkpoint}
```
Comment (MegaRedHand) [inline bench/ssz/ssz_ex_bench.exs:42]:
```suggestion
    "BeaconState" => state,
    "Checkpoint" => checkpoint
```

**Review by MegaRedHand — COMMENTED** [review]
LGTM. @mpaulucci was this what you had in mind?

File: bench/ssz/merkleize_bench.exs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+alias LambdaEthereumConsensus.SszEx
```
Comment (mpaulucci) [inline bench/ssz/merkleize_bench.exs:1]: I would name this one hash_tree_root which is more important than the merkelization part

**Review by mpaulucci — APPROVED** [review]

**Pull request merged**